packages/math-input/src/components/provided-keypad.tsx

```diff
@@ -24,21 +23,13 @@ type Props = {
 };
 
 class ProvidedKeypad extends React.Component<Props> {
-    mounted?: boolean;
     store: any;
 
-    UNSAFE_componentWillMount() {
+    constructor(props) {
+        super(props);
         this.store = createStore();
     }
```

Comment:
I couldn't see where `mounted` was ever getting used?

And I can change it back, but it felt like we could just use the constructor instead of a depreciated lifecycle.

Reply:
In other parts of Perseus, we use a flag like this in cases where we defer something with a timeout.  I see there's a setTimeout in `configure()`.  That doesn't look like it'd need a `mounted` guard though as we don't call into code that touches the component.  So it's probably something that was refactored at some point and the `mounted` flag just wasn't cleaned up.  👍